tests/Propel/Tests/Issues/Issue1659Test.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Tests\Issues;
+
+use Propel\Generator\Util\QuickBuilder;
+use Propel\Tests\TestCaseFixtures;
+
+/**
+ * Test : Propel should not allow incomplete foreign key references when foreign table has a composite primary key
+ * Issue described in https://github.com/propelorm/Propel2/issues/675.
+ * Originally described in http://stackoverflow.com/questions/7947085/are-incomplete-key-references-in-propel-useful
+ *
+ * @group model
+ */
+class Issue1659Test extends TestCaseFixtures
```

Comment:
We need to find better name for this class

Reply:
Sure, that can be done as follow up. For now I guess people find the details when needed.